release-habushu.sh

```diff
@@ -41,8 +48,7 @@ if [[ "$?" -ne 0 ]] ; then
 fi
 
 echo "///////////  Commit all changes that reflect the target release version and create a tag ///////////"
-mvn scm:checkin -Dmessage="Prepare release habushu-$1"
-mvn scm:tag -Dtag=habushu-$1
+mvn scm:checkin -Dmessage="Prepare release habushu-$1" && mvn scm:tag -Dtag=habushu-$1
```

Comment:
I: Technically because these were two separate commands, the exit code check below only checked if the tag was successful and not if the checkin was successful.